src/AddTodoForm.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+let AddTodoForm = () => {
+    return (
+        <div>
+            <form>
+                <label htmlFor="todoTitle">Title:</label><br />
+                <input type="text" id="todoTitle" required /><br />
+            </form>
+            <button>Add</button>
```

Comment:
What `type` of button do we have here?

Reply:
There is also another bug that's a bit hard to spot since you can't submit anything with this form yet. I would reference the examples in the reading and see if you can spot it.